File: points_of_interest/v1/by_square/ios/PointsOfInterest.swift
import Amadeus

var amadeus: Amadeus = Amadeus(
    client_id: "YOUR_API_ID",
    client_secret: "YOUR_API_SECRET"
)

// What are the popular places in Barcelona? (based on a square)
amadeus.referenceData.locations.pointsOfInterest.bySquare.get(params: ["north": "41.397158",
                                                                       "west": "2.160873",
                                                                       "south": "41.394582",
                                                                       "east": "2.177181"],
                                                              onCompletion: {
                                                                  response, error in
                                                                  if error == nil {
                                                                      print(response!.data)
                                                                  }
    })

File: points_of_interest/v1/by_square/node/points_of_interest.js
var Amadeus = require("amadeus");
var amadeus = new Amadeus({
  clientId: 'YOUR_API_KEY',
  clientSecret: 'YOUR_API_SECRET'
});

// What are the popular places in Barcelona? (based on a square)
amadeus.referenceData.locations.pointsOfInterest.bySquare.get({
  north: 41.397158,
  west: 2.160873,
  south: 41.394582,
  east: 2.177181
}).then(function (response) {
  console.log(response);
}).catch(function (response) {
  console.error(response);
});